src/styles/components/_product-card.scss

```diff
@@ -0,0 +1,99 @@
+.product-card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid $color-grey-light;
+  background: $color-white;
+  font-family: Roboto, Helvetica, sans-serif;
+  transition: all 0.3s;
+
+  &__picture {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    color: $color-main-accent;
+    line-height: 18px;
+
+  }
+
+  &:hover {
+    transform: scale(1.2);
+
+    & .product-card__name {
+      color: $color-hover-accent;
+    }
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    color: $color-secondary;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px; /* 140% */
+  }
+
+  &__reviews {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
```

Comment:
will be better move these properties to `mixins` and reuse them

Reply:
Created and used mixin flex-center